packages/create-corvid-app/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "create-corvid-app",
+  "version": "0.1.0",
+  "description": "Bootstrap local corvid apps with just one command",
```

Comment:
@wixysam wdyt about this sentence ?

Reply:
We use "local site project" instead of "local corvid apps" in the documentation. Probably makes sense to use that here as well. Also, I think I'm fine with the word "bootstrap" for this because we are assuming a more technical audience than standard Corvid stuff, but I think it should be cleared with product.